src/testFixtures/java/uk/gov/di/ipv/cri/common/library/aws/CloudFormationHelper.java

```diff
@@ -15,7 +15,13 @@ public final class CloudFormationHelper {
 
     private CloudFormationHelper() {}
 
-    /** Get a parameter value from a stack */
+    /**
+     * Get a parameter value from a stack
+     *
+     * @deprecated as new test harness implementation allows us to assert that TxMA events written
+     *     by CRIs and supports automated testing of the CloudFront headers
+     */
+    @Deprecated(since = "4.0.0")
```

Comment:
We can remove the deprecated annotation on getParameter as this can used independently from the SQSHelper